source/guide_magicmirror.rst

```diff
@@ -0,0 +1,120 @@
+.. highlight:: console
+
+.. author:: Thomas Kammerer <https://kammerer.tk/>
+
+.. tag:: lang-nodejs
+.. tag:: web
+.. tag:: smart-mirror
+
+.. sidebar:: Logo
+
+  .. image:: _static/images/magicmirror.png
+      :align: center
+
+############
+MagicMirror²
+############
+
+.. tag_list::
+
+MagicMirror²_ is an open source modular smart mirror platform. With a growing list of installable modules, the MagicMirror² allows you to convert your hallway or bathroom mirror into your personal assistant. MagicMirror² is built by the creator of the original MagicMirror with the incredible help of a growing community of contributors.
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * :manual:`Node.js <lang-nodejs>` and its package manager :manual_anchor:`npm <lang-nodejs.html#npm>`
+  * :manual:`supervisord <daemons-supervisord>`
+  * :manual:`domains <web-domains>`
+
+Prerequisites
+=============
+
+Node and npm
+------------
+
+We're using :manual:`Node.js <lang-nodejs>` in the latest version:
+
+.. code-block:: console
+
+ [isabell@stardust ~]$ uberspace tools version show node
+ Using 'Node.js' version: '12'
+ [isabell@stardust ~]$
+
+Your website domain needs to be set up:
+
+.. include:: includes/web-domain-list.rst
+
+Installation
+============
+
+We clone the repository to our home directory and install the application.
+
+.. code-block:: console
+
+ [isabell@stardust ~]$ git clone https://github.com/MichMich/MagicMirror
+ [isabell@stardust ~]$ cd MagicMirror
+ [isabell@stardust MagicMirror]$ npm install
+ […]
+ [isabell@stardust MagicMirror]$ npm audit fix --force
+ […]
+ [isabell@stardust MagicMirror]$
+
+Configuration
+=============
+
+After the installation you need to setup your config file.
+
+.. code-block:: console
+
+ [isabell@stardust MagicMirror]$ cp config/config.js.sample config/config.js
+ [isabell@stardust MagicMirror]$
+
+Change the address from ``localhost`` to ``0.0.0.0``. You may remove the IP addresses in ``ipWhitelist`` to access your MagicMirror² from everywhere.
+
+.. code-block:: javascript
+ :emphasize-lines: 1
+
+      address: "0.0.0.0",
+
+Setup daemon
+------------
+
+Create ``~/etc/services.d/magicmirror.ini`` with the following content:
+
+.. code-block:: ini
+
+ [program:magicmirror]
+ directory=%(ENV_HOME)s/MagicMirror
+ command=npm start
+ autostart=yes
+ autorestart=yes
+
+.. include:: includes/supervisord.rst
+
+If it's not in state RUNNING, check your configuration.
+
+Configure web server
+--------------------
+
+.. note::
+
+    MagicMirror² running on port 8080.
+
+.. include:: includes/web-backend.rst
+
+Updates
+-------
+
+.. note:: Check the update feed_ regularly to stay informed about the newest version.
+
+Stop your service and repeat the installation step.
+
+.. _MagicMirror²: https://magicmirror.builders/
+.. _feed: https://github.com/MichMich/MagicMirror/releases
+
+----
+
```

Comment:
I can't get it to work 😞

```
[moritz@stardust MagicMirror]$ npm start
npm WARN lifecycle The node binary used for scripts is /bin/node but npm is using /opt/nodejs12/bin/node itself. Use the `--scripts-prepend-node-path` option to include the path for the node binary npm was executed with.

> magicmirror@2.11.0 start /home/moritz/MagicMirror
> DISPLAY="${DISPLAY:=:0}" ./node_modules/.bin/electron js/electron.js

/home/moritz/MagicMirror/node_modules/electron/dist/electron: error while loading shared libraries: libgtk-3.so.0: cannot open shared object file: No such file or directory
npm ERR! code ELIFECYCLE
npm ERR! syscall spawn
npm ERR! file sh
npm ERR! errno ENOENT
npm ERR! magicmirror@2.11.0 start: `DISPLAY="${DISPLAY:=:0}" ./node_modules/.bin/electron js/electron.js`
npm ERR! spawn ENOENT
npm ERR! 
npm ERR! Failed at the magicmirror@2.11.0 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/moritz/.npm/_logs/2020-04-28T08_20_54_802Z-debug.log
```

Is it only me?

Reply:
Just fixed it, the command to start the server is `npm run server` not `npm start`.